### DISCUSSION TITLE
Something went wrong with my badges of readme

### DISCUSSION THREAD

**Discussion by Sagar-Sharma-7** [community/community#35207] [Other Feature Feedback, Questions, & Ideas]
<img width="650" alt="image" src="https://user-images.githubusercontent.com/78422995/194100295-303e3bdf-31fe-4428-aaef-5f746a49c4f7.png">

**Comment by Sabyasachi-Seal** [comment]
Dont worry this happend with all of us. The app basically got rate limited. Now the issue is fixed as the creator has increased the max calls to the api.

**Comment by jfullstackdev** [comment]
yup, I'm using the external service too, most of the time it will return normally
